p2p/protocol/identify/id.go

```diff
@@ -31,9 +33,27 @@ const ID = "/ipfs/id/1.0.0"
 
 // LibP2PVersion holds the current protocol version for a client running this code
 // TODO(jbenet): fix the versioning mess.
+// XXX: Don't change this till 2020. You'll break all go-ipfs versions prior to
+// 0.4.17 which asserted an exact version match.
 const LibP2PVersion = "ipfs/0.1.0"
 
-var ClientVersion = "go-libp2p/3.3.4"
+// ClientVersion is the default user agent.
+//
+// Deprecated: Set this with the UserAgent option.
+var ClientVersion = "github.com/libp2p/go-libp2p"
```

Comment:
let's just call this `go-libp2p`.

Reply:
Sure but users using go mod will still use their full paths by default. E.g., `github.com/libp2p/go-libp2p-examples@v0.1.1`.

Reply:
oh ok, nevermind then.

Reply:
This will increase the size of the identify messages by 15 bytes. It's ok but worth noting.